### PR TITLE
MES-683 - pass / fail links removes

### PR DESCRIPTION
**Pull request by a0dexx**
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/test-report/cat-c/test-report.cat-c.page.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <ion-header>
   <ion-navbar hideBackButton>
     <ion-title>Test report - {{pageState.candidateUntitledName$ | async}}</ion-title>
-    <ion-buttons start>
-      <button ion-button (click)="passTest()" >Pass Test</button>
-      <button ion-button (click)="failTest()" >Fail Test</button>
-    </ion-buttons>
+<!--    <ion-buttons start>-->
```
Comment (TomBillingtonUK) [inline src/pages/test-report/cat-c/test-report.cat-c.page.html]: Can we delete the code rather the comment it out

**Review by TomBillingtonUK — APPROVED** [review]
Looks good

**Review by MarkPatt89 — APPROVED** [review]
Looks good, agree with Tom about deletion rather than commenting.

File: src/pages/test-report/cat-c/test-report.cat-c.page.html (hunk 1 above)
Comment (edwardwilson) [inline src/pages/test-report/cat-c/test-report.cat-c.page.html]: Is there a reason this is commented out rather than just removed?

**Review by edwardwilson — APPROVED** [review]

**Pull request merged**